pallets/pools/src/lib.rs

```diff
@@ -120,6 +130,10 @@ where
 	pub min_epoch_time: Moment,
 	pub challenge_time: Moment,
 	pub max_nav_age: Moment,
+	pub min_update_delay: Moment,
+	pub require_redeem_fulfillments_before_updates: bool,
+	pub scheduled_update: Option<PoolUpdate<Rate>>,
+	pub scheduled_update_executed_after: Option<Moment>,
```

Comment:
I would prefer to have a seperate struct for this if not change the logic to have a separate storage for this. 
So wrapping the new update fields into a `struct UpdateDetails`

Reply:
I think we can split this. Having the `min_update_delay` and the requirements as parts of an `UpdateDetails` struct and the scheduled infos as a separate storage entry. I don`t wanna bloat the `PoolDetails` more.